### PR TITLE
Surfacing events for pods created by config files. 

### PR DESCRIPTION
**Pull request by dchen1107**
Also add column Source to the output of 'kubectl get events' so that we know the events are from node.

### CODE REVIEW

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -481,6 +481,7 @@ func fieldPath(pod *api.BoundPod, container *api.Container) (string, error) {
 // system.
 func containerRef(pod *api.BoundPod, container *api.Container) (*api.ObjectReference, error) {
 	fieldPath, err := fieldPath(pod, container)
+	glog.Infof("???????FieldPath: %s", fieldPath)
```
Comment (erictune) [inline pkg/kubelet/kubelet.go]: Remove if for debugging or make more descriptive.

File: pkg/kubelet/util.go [hunk 2]
```diff
@@ -130,8 +130,9 @@ func SetupEventSending(authPath string, apiServerList util.StringList) {
 			glog.Errorf("Unable to make apiserver client: %v", err)
 		} else {
 			// Send events to APIserver if there is a client.
+			hostname := util.GetHostname("")
 			glog.Infof("Sending events to APIserver.")
-			record.StartRecording(apiClient.Events(""), "kubelet")
+			record.StartRecording(apiClient.Events(""), "kubelet:"+hostname)
```
Comment (erictune) [inline pkg/kubelet/util.go]: We just talked about adding a new field for hostname of the component.

**Comment by dchen1107** [comment]
Addressed comments. PTAL?

**Comment by dchen1107** [comment]
Travis issue is fixed by #3256 now.

**Pull request merged**